tests/attention/test_vsa_block_sparse.py

```
@@ -0,0 +1,1076 @@
+            indptr, indices = _build_random_bsr(num_blocks, num_blocks, density, device)
+            active_blocks = len(indices)
+
+            wrapper = _make_wrapper_blk64(device)
+            wrapper.plan(
+                indptr, indices, seqlen, seqlen, R64, C64,
+                num_heads, num_heads, HEAD_DIM_BLK64,
+                q_data_type=dtype,
+            )
+            wrapper.run(q, k, v)  # warm-up
+
+            times = bench_gpu_time(lambda: wrapper.run(q, k, v))
```

Ruff (0.15.15)

[warning] Function definition does not bind loop variable `wrapper`

(B023)